Makefile

```diff
@@ -378,12 +378,20 @@ docker-up:
 docker-up-no-build: docker-up-db
 	$(DOCKER_COMPOSE) up -d --scale koku-worker=$(scale)
 
-docker-up-min:
+# basic dev environment targets
+docker-up-min: docker-up-db
 	$(DOCKER_COMPOSE) build koku-base
-	$(DOCKER_COMPOSE) up -d --scale koku-worker=$(scale) db redis koku-server masu-server koku-worker
+	$(DOCKER_COMPOSE) up -d --scale koku-worker=$(scale) redis koku-server masu-server koku-worker
 
 docker-up-min-no-build: docker-up-db
-	$(DOCKER_COMPOSE) up -d --scale koku-worker=$(scale) redis koku-server masu-server koku-worker koku-listener
+	$(DOCKER_COMPOSE) up -d --scale koku-worker=$(scale) redis koku-server masu-server koku-worker
+
+# basic dev environment targets with koku-listener for local Sources Kafka testing
+docker-up-min-with-kafka: docker-up-min docker-up-db
```

Comment:
suggestion (blocking): let's change these to `with-listener` since these commands do not start kafka.

Reply:
Good point
Done